### PR TITLE
Create components

### PR DESCRIPTION
**Pull request by MahdiSohaily**
### Book Store Project: Create reusable components
in this branch I have made the following changes:

- Added react route to the project.
- Added project-specified pages.
- Added project-related components.

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @MahdiSohaily, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no professional Documentations, kindly consider using the README template provided by microverse
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/BookList.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/jsx-key */
```
Comment (cynthiainga) [inline src/components/BookList.tsx:1]: - [ ] Please kindly do not disable linters, fix linters errors helps programmers to write clean codebase 😁 😉

File: src/pages/Categories.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React, { ReactElement } from 'react'
+
+export default function Categories (): ReactElement {
+  return (
+    <div>Categories</div>
```
Comment (cynthiainga) [inline src/pages/Categories.tsx:5]: - [ ] Categories should display only a button with the text "Check status" is not the tex`Categories` 😁😁

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @MahdiSohaily, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 
- [ ] Correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Categories.tsx (hunk 2 above)
Comment (rloterh) [reply src/pages/Categories.tsx]: This issue is well resolved! ✔️

File: src/components/BookList.tsx (hunk 1 above)
Comment (rloterh) [reply src/components/BookList.tsx]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Mahdi,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾Congratulations!  🎉💯🌟 

## Highlights 👍
- Good job implementing recommended changes

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- It's recommended that you add an MIT license or any other similar license to your ReadMe for this project.
Implementing the above change(s) will make your ReadMe/ repository stand out even better 😉

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**